velox/docs/functions/presto/coverage.rst

```diff
@@ -2,7 +2,7 @@
 Function Coverage
 =================
 
-Here is a list of all scalar and aggregate Presto functions with functions that are available in Velox highlighted.
+Here is a list of all scalar, aggregate, and window functions from Presto, with functions that are available in Velox highlighted.
```

Comment:
@pramodsatya : Can you generate the doc and check the end result. The viewer is showing a strange part with an entry for not
![image](https://github.com/user-attachments/assets/e1448446-0098-4f31-9119-31905b68d57e)